flipt-rust/src/api/mod.rs

```diff
@@ -44,6 +37,6 @@ impl FliptClient {
 
 impl Default for FliptClient {
     fn default() -> Self {
-        Self::new(Config::default()).unwrap()
+        Self::new::<ClientTokenAuthentication>(Config::default()).unwrap()
```

Comment:
I think we might want a `None` authentication strategy and make that the default?

Reply:
@markphelps Haha yeah that was what I was thinking. Should have posed that question here. I added it.